src/index.html

```diff
@@ -1,15 +1,200 @@
+              type="text"
+              name="name"
+              id="Name"
+              required
+              autocomplete="off"
+            >
+          </label>
+        </div>
+        <div class="block">
+          <label for="Age">
+            How old are you?
+            <input
+              type="number"
+              name="age"
+              id="Age"
+              min="12"
+              max="100"
+              value="12"
+            >
+          </label>
+        </div>
+        <div class="block">
+          <label for="Date">
+            Full date of birth:
+            <input
+              type="date"
+              name="date"
+              id="Date"
+              required
+            >
+          </label>
+        </div>
+        <div class="block">
+          <label for="Acception">
+            I accept the term of the agreement:
+            <input
+              type="checkbox"
+              name="acception"
+              id="Acception"
+              required
+            >
+          </label>
+        </div>
+      </fieldset>
+      <fieldset>
+        <legend>Registration</legend>
+        <div class="block">
+          <label for="Email">
+            E-mail:
+            <input
+              type="email"
+              name="email"
+              id="Email"
+              placeholder="email@example.com"
+              required
+            >
+          </label>
+        </div>
+        <div class="block">
+          <label for="Password">
+            Password:
+            <input
+              type="password"
+              name="password"
+              id="Password"
+              minlength="5"
+              maxlength="16"
+              required
+            >
+          </label>
+        </div>
+      </fieldset>
+      <fieldset>
+        <legend>An interesting fact about you!</legend>
+        <div class="block">
+          Do you love cats?
+            <input
+              type="radio"
+              name="cats"
+              id="Yes">
+          <label for="Yes">
+            Yes
+          </label>
+            <input
+              type="radio"
+              name="cats"
+              id="No">
+          <label for="No">
+            No
+          </label>
+        </div>
+        <div class="block">
+          <label for="Color">
+            What is your favorite color?
+            <input
+              type="color"
+              name="color"
+              id="Color"
+            >
+          </label>
+        </div>
+        <div class="block">
+          <label for="Bedtime">
+            What time do you go to bed?
+            <input
+              type="time"
+              name="bedtime"
+              id="Bedtime"
+            >
+          </label>
+        </div>
+        <div class="block">
+          <label for="Cars">
+            What are your favorite brand of cars?
+            <select
+              name="cars"
+              id="Cars"
+              multiple>
```

Comment:
don't forget to wrap a closing bracket
```suggestion
              multiple
            >
```